config/change_branch.sh

```diff
@@ -0,0 +1,42 @@
+export CODEBUILD_GIT_BRANCH="$(git symbolic-ref HEAD --short 2>/dev/null)"
+if [ "$CODEBUILD_GIT_BRANCH" = "" ] ; then
+  CODEBUILD_GIT_BRANCH="$(git branch -a --contains HEAD | sed -n 2p | awk '{ printf $1 }')";
+  export CODEBUILD_GIT_BRANCH=${CODEBUILD_GIT_BRANCH#remotes/origin/};
+fi
+
+cd $CODEBUILD_SRC_DIR && git checkout $CODEBUILD_GIT_BRANCH
+export CURRENT_COMMIT_HASH=$(git log -1 --pretty=%h);
+#export CURRENT_COMMIT_DATE="$(git show -s --format=%ci | cut -d' ' -f 1)$(git show -s --format=%ci | cut -d' ' -f 2)";
+export CURRENT_DATETIME=$(date +'%Y%m%d_%H%M%S')
+export CURRENT_REPO_NAME=$(basename `git rev-parse --show-toplevel`) ;
+export CURRENT_COMMIT_PATH="$CURRENT_DATETIME/$CURRENT_COMMIT_HASH"
+cd ..
```

Comment:
```suggestion
popd
```
safer to do `pushd..popd` if some commands in between change the working directory

Reply:
Same as above.